modules/nlu/src/backend/predict-pipeline.ts

```diff
@@ -493,8 +505,9 @@ function MapStepToOutput(step: PredictStep, startTime: number): PredictOutput {
         confidence: s.confidence,
         name: s.name,
         source: s.source,
-        value: s.value
-      } as sdk.NLU.Slot
+        value: s.value,
+        entity: entitiesMapper(s.entity)
```

Comment:
can you add a comment so we don't forget to include this in legacy intent election ? in dev branch when we merge ?

Reply:
sure!